src/internal/compareLength.js

```diff
@@ -0,0 +1,8 @@
+import { curry, compose, flip, length } from 'ramda';
+
+const compareLength = comparator =>
```

Comment:
Pls do the changes that were discussed in https://github.com/char0n/ramda-adjunct/issues/527